jest.transform.js

```diff
@@ -5,7 +5,8 @@ const babelOptions = {
       {
         targets: {
           node: 14
-        }
+        },
+        loose: true
```

Comment:
@ojeytonwilliams Does #42234  resolve this?

Reply:
It should do, yes.